src/git-lfs/install.sh

```diff
@@ -82,7 +82,7 @@ receive_gpg_keys() {
         echo "(*) Downloading GPG key..."
         ( echo "${keys}" | xargs -n 1 gpg --recv-keys) 2>&1 && gpg_ok="true"
         if [ "${gpg_ok}" != "true" ]; then
-            echo "(*) Failed getting key, retring in 10s..."
+            echo "(*) Failed getting key, retrying in 10s..."
```

Comment:
Can we also bump patch versions for all the updated Features? Thank you!

Reply:
Certainly, I've addressed this in [500562b](https://github.com/devcontainers/features/pull/1006/commits/500562b6394645cc4f7db3d9a3f027224da07ab5)